sql/sqlite/inspect.go

```diff
@@ -715,7 +727,7 @@ const (
 	databasesQuery     = "SELECT `name`, `file` FROM pragma_database_list() WHERE `name` <> 'temp'"
 	databasesQueryArgs = "SELECT `name`, `file` FROM pragma_database_list() WHERE `name` IN (%s)"
 	// Query to list database tables.
-	tablesQuery = "SELECT `name`, `sql` FROM sqlite_master WHERE `type` = 'table' AND `name` NOT LIKE 'sqlite_%' AND `name` NOT LIKE 'libsql_%'"
+	tablesQuery = "SELECT sqlite_master.name, sqlite_master.sql, `wr`, `strict` FROM sqlite_master join(pragma_table_list(sqlite_master.name)) WHERE sqlite_master.type = 'table' AND sqlite_master.name NOT LIKE 'sqlite_%' AND sqlite_master.name NOT LIKE 'libsql_%'"
```

Comment:
The query became a bit complex, add line breaks.
```suggestion
	tablesQuery = `
SELECT
	sqlite_master.name, sqlite_master.sql, wr, strict
FROM
	sqlite_master
	JOIN pragma_table_list(sqlite_master.name)
WHERE
	sqlite_master.type = 'table'
	AND sqlite_master.name NOT LIKE 'sqlite_%'
	AND sqlite_master.name NOT LIKE 'libsql_%'`
```